ods-provisioning-app/ocp-config/cm.yml

```diff
@@ -237,6 +237,9 @@ objects:
       openshift.jenkins.project.name.pattern=jenkins-%s-cd%s
       openshift.jenkins.project.webhookproxy.host.pattern=webhook-proxy-%s-cd%s
 
+      # list of supported webhook events
+      openshift.jenkins.project.webhookproxy.events=repo:refs_changed,pr:merged,pr:declined,pr:deleted
```

Comment:
Can we extract the value to a parameter? That way one can override it - `pr:opened` is also supported, but not added by default.

The parameter should then end up in the `configuration-sample/ods-core.env.sample` file.

Reply:
yeah... it is added now in the 2nd commit